### PR TITLE
[Aftershock] misc typograpical fixes

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
To fix some small typographical errors, note that I am not a native English speaker, (I just happen to have a file with all the aftershock descriptions, vim's handy spell checker and the internet for references) so apologies for possible obvious mistakes.

### CODE REVIEW

File: data/mods/Aftershock/player/professions.json [hunk 1]
```diff
@@ -192,7 +192,7 @@
     "id": "afs_wraitheon_executive",
     "name": "Wraitheon Executive",
     "description": {
-      "str": "You were one of the chief executives of the Megacorporation Wraitheon Robotics ltd. and consistently ranked among the most powerful and influential persons of the world.  Still, the apocalypse has caught you unprepared, and now after the dissolution of your company and the crumbling of your network of influence, your were left powerless and destitute, and must rely on your two robotic bodyguards to survive.",
+      "str": "You were one of the chief executives of the Megacorporation Wraitheon Robotics ltd. and consistently ranked among the most powerful and influential persons of the world.  Still, the apocalypse has caught you unprepared, and now after the dissolution of your company and the crumbling of your network of influence, you were left powerless and destitute, and must rely on your two robotic bodyguards to survive.",
```
Comment (actual-nh) [inline data/mods/Aftershock/player/professions.json:195]:
```suggestion
      "str": "You were one of the chief executives of the Megacorporation Wraitheon Robotics ltd. and consistently ranked among the most powerful and influential persons of the world.  Still, the apocalypse has caught you unprepared, and now after the dissolution of your company and the crumbling of your network of influence, you are left powerless and destitute, and must rely on your two robotic bodyguards to survive.",
```

**Review by actual-nh — COMMENTED** [review]
A few extra things; good spotting! (I have tried to avoid purely stylistic changes.)

**Review by Maleclypse — APPROVED** [review]
These looked good to me.

File: data/mods/Aftershock/items/comestibles/alienfood.json [hunk 2]
```diff
@@ -254,7 +254,7 @@
     "copy-from": "alien_stomach",
     "type": "COMESTIBLE",
     "name": "large alien stomach",
-    "description": "A balloon like organ of a large alien.  It is surprisingly durable.",
+    "description": "A balloon-like organ of a large alien.  It is surprisingly durable.",
```
Comment (ZhilkinSerg) [inline data/mods/Aftershock/items/comestibles/alienfood.json:257]:
```suggestion
    "description": "A balloon-like organ of a large alien you can only guess is a stomach.  It is surprisingly durable.",
```

**Pull request merged**